packages/fast-components-react-msft/app/examples.ts

```diff
@@ -1,2 +1,4 @@
 import ButtonExamples from "../src/button/examples.data";
-export {ButtonExamples};
+import MediaExamples from "../src/media/examples.data";
+
+export {ButtonExamples, MediaExamples};
```

Comment:
Same comment as in react-base. Let's keep these separate so they don't get out of hand.